course.go

```diff
@@ -50,11 +42,9 @@ type CourseFindUpdater interface {
 }
 
 type CourseRepository interface {
-	CourseInserter
```

Comment:
Why did you remove this one and not `CourseMemberAdder` and `CourseMemberRemover`?